ui/v2.5/src/index.scss

```diff
@@ -465,26 +465,72 @@ div.dropdown-menu {
 .top-nav {
   padding: 0.25rem 1rem;
 
+  .navbar-toggler {
+    @include media-breakpoint-down(xs) {
+      @media (orientation: portrait) {
+        bottom: 50px;
+        position: fixed;
+        right: 25px;
+
+        &::before {
+          background: #137cbd;
+          border-radius: 50%;
+          content: "";
+          left: 50%;
+          padding-top: 125%;
+          position: absolute;
+          top: 50%;
+          transform: translate(-50%, -50%);
+          width: 125%;
+          z-index: -1;
+        }
+      }
+    }
+  }
+
+  .navbar-collapse {
+    max-height: calc(100vh - 4rem);
+    overflow-y: scroll;
```

Comment:
With tablet emulation a scrollbar is shown as well during the transition. Not sure if it will be shown in actual mobile browsers though. But IMO it shouldn't be the case that there is a scrollbar. (Overflow should only be enabled on screen sizes where the height is less than 400px or whatever I guess. Which should be doable using a media query as well using `min-height`)